config/alpha.json

```diff
@@ -13,10 +13,11 @@
   "leoUrlRoot": "https://leonardo.dsde-alpha.broadinstitute.org",
   "marthaUrlRoot": "https://us-central1-broad-dsde-alpha.cloudfunctions.net",
   "orchestrationUrlRoot": "https://firecloud-orchestration.dsde-alpha.broadinstitute.org",
+  "profileUrlRoot": "https://profile-dot-broad-shibboleth-prod.appspot.com/dev",
   "rawlsUrlRoot": "https://rawls.dsde-alpha.broadinstitute.org",
   "rexUrlRoot": "https://terra-rex-alpha.appspot.com",
   "samUrlRoot": "https://sam.dsde-alpha.broadinstitute.org",
-  "shibbolethUrlRoot": "http://mock-nih.dev.test.firecloud.org",
+  "shibbolethUrlRoot": "https://broad-shibboleth-prod.appspot.com/dev",
```

Comment:
Unrelated to PR: The URL here seems super weird because of the prod in the URL host. If we take over shibboleth, I might push to see if we can change that.

Reply:
Agree, the current approach is to have both dev and prod co-located, but I think it would be better to split them up.